src/components/how-to-play.js

```diff
@@ -47,14 +47,17 @@ export class HowToPlay extends Screen {
     }
 
     setButtonVisibility() {
-        this.buttonLayout.buttons.previous.visible = Boolean(!this.theme.howToPlay || this.currentIndex !== 0);
+        this.layout.buttons.previous.visible = Boolean(!this.theme.howToPlay || this.currentIndex !== 0);
 
         const isNotLastPage = this.currentIndex + 1 !== this.choiceSprites.length;
-        this.buttonLayout.buttons.next.visible = Boolean(!this.theme.howToPlay || isNotLastPage);
+        this.layout.buttons.next.visible = Boolean(!this.theme.howToPlay || isNotLastPage);
+
+        this.layout.buttons.previous.accessibleElement.update();
```

Comment:
Do we need tests for these new lines?

Reply:
Visibility is tested in the fixed tests below.
Testing update is called feels like the wrong thing to check. Will have a think...

Reply:
Ended up testing update is called.
Not sure that's the best thing but it's mocked so prob all that can be done.